docs/faq.md

```diff
@@ -40,14 +40,14 @@ However, if you run Yazi in WSL, you can experience perfect image previews using
 
 You need to install `sshd` in WSL and start it:
 
-```bash
+```shell
```

Comment:
Not sure why, but `shell` is not working on my neovim...

![CleanShot 2024-01-11 at 22 25 07](https://github.com/yazi-rs/yazi-rs.github.io/assets/17523360/31ea3684-997d-47e9-9d81-726b38b2b9fa)

Reply:
That sounds like a Neovim issue!
![Screenshot 2024-01-11 at 09 29 38 (Arc)](https://github.com/yazi-rs/yazi-rs.github.io/assets/47499684/a269361a-c0a9-40b1-9b76-2ac2c4b9acc2)
![Screenshot 2024-01-11 at 09 30 11 (Visual Studio Code)](https://github.com/yazi-rs/yazi-rs.github.io/assets/47499684/6564d0b6-4453-44a0-9e3a-c9c03c8421ae)

Reply:
But `sh` works (also works in Docusaurus)! How about using `sh` instead of `shell`?
![CleanShot 2024-01-11 at 22 33 31](https://github.com/yazi-rs/yazi-rs.github.io/assets/17523360/2e12965a-96fb-4a1b-b422-f7bd8cf13458)

Reply:
Okay that is a Neovim issue ahah. Sure I guess... you should make an issue for whatever Neovim plugins you use.

Reply:
I found this, https://github.com/MDeiml/tree-sitter-markdown/issues/42, it seems that Neovim just doesn't support shell yet and it's not a bug

Reply:
If it doesn't support it yet it kinda is a bug ahah.